### PR TITLE
 tracking more states

### PR DESCRIPTION
**Pull request by arnitolog**
with this PR we will be able to track any task state. Not only "running" and/or "failed"

### CODE REVIEW

File: kafka_connect_exporter.go [hunk 1]
```diff
@@ -136,13 +136,19 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
 
 		for _, connectorTask := range connectorStatus.Tasks {
 
-			var isTaskRunning float64 = 0
+			var taskState float64 = 0
 			if strings.ToLower(connectorTask.State) == "running" {
-				isTaskRunning = 1
+				taskState = 1
+			}
+			if strings.ToLower(connectorTask.State) == "unassigned" {
```
Comment (wakeful) [inline kafka_connect_exporter.go]: switch expression?

File: kafka_connect_exporter.go [hunk 2]
```diff
@@ -136,13 +136,19 @@ func (e *Exporter) Collect(ch chan<- prometheus.Metric) {
...
+				taskState = 2
+			}
+			if strings.ToLower(connectorTask.State) == "paused" {
+				taskState = 3
 			}
 
 			ch <- prometheus.MustNewConstMetric(
-				areConnectorTasksRunning, prometheus.GaugeValue, isTaskRunning,
+				areConnectorTasksRunning, prometheus.GaugeValue, taskState,
```
Comment (wakeful) [inline kafka_connect_exporter.go]: can you also update the `HELP` section for the [metric](https://github.com/wakeful/kafka_connect_exporter/blob/master/kafka_connect_exporter.go#L36) (and Readme file)?

**Review by wakeful — CHANGES_REQUESTED** [review]
.

File: kafka_connect_exporter.go (hunk 1 above)
Comment (arnitolog) [reply kafka_connect_exporter.go]: yeah. makes sense. fixed

File: kafka_connect_exporter.go (hunk 2 above)
Comment (arnitolog) [reply kafka_connect_exporter.go]: updated. please take a look

**Review by wakeful — APPROVED** [review]

**Pull request merged**